app/distributions/api.py

```diff
@@ -17,30 +16,30 @@
 router = Router()
 
 
-def attribution_to_response(model: Attribution, lang: LanguageCode) -> AttributionSchema:
+def attribution_to_response(model: Attribution, lang: LanguageCode) -> dict:
     """
-    Transforms the given model using the given language into a response object.
+    Transforms the given model using the given language into the response structure
     """
-    response = AttributionSchema(
-        id=str(model.id),
-        name=get_translation(model, "name", lang),
-        name_translations=TranslationsSchema(
-            de=model.name_de,
-            fr=model.name_fr,
-            en=model.name_en,
-            it=model.name_it,
-            rm=model.name_rm,
-        ),
-        description=get_translation(model, "description", lang),
-        description_translations=TranslationsSchema(
-            de=model.description_de,
-            fr=model.description_fr,
-            en=model.description_en,
-            it=model.description_it,
-            rm=model.description_rm,
-        ),
-        provider_id=str(model.provider.id),
-    )
+    response = {
+        "id": str(model.id),
+        "name": get_translation(model, "name", lang),
+        "name_translations": {
+            "de": model.name_de,
```

Comment:
As a demonstration that Ninja handles the transformation, remove the field "de" here and Ninja throws a `ValidationError`

Reply:
I understand but I don't see the advantage of using a dict over a schema here.

With a structured object like the schema, we have these advantages:
1. Clearer interface: It returns an object with a defined set of fields, not just a dict that could contain anything.
2. Facilitates type checking early: The type checker (mypy) can already find issues at build time instead of at runtime.
3. Easier coding: The IDE can suggest the field names and assist in renaming fields everywhere they are used.

What speaks in favor of using a dict instead?